### ISSUE TITLE
Set up redis

### ISSUE THREAD

**Issue by keighrim** [keighrim/cs105_nT#57]
from Pito: [REDIS] Investigate and figure out how to setup a Redis Caching service

**Comment by keighrim** [comment]
Tests fails after redis integration. Below is errors from codeship.

```
./home/rof/src/github.com/keighrim/cs105_nT/models/follow.rb:11: warning: global variable `$redis' not initialized
E/home/rof/src/github.com/keighrim/cs105_nT/models/follow.rb:11: warning: global variable `$redis' not initialized
E./home/rof/src/github.com/keighrim/cs105_nT/models/tweet.rb:16: warning: global variable `$redis' not initialized
E./home/rof/src/github.com/keighrim/cs105_nT/models/tweet.rb:16: warning: global variable `$redis' not initialized
E/home/rof/src/github.com/keighrim/cs105_nT/models/tweet.rb:16: warning: global variable `$redis' not initialized
E
```

Any clue?

**Comment by keighrim** [comment]
Nevertheless, the app seems to run fine.

**Comment by ACHEZ9** [comment]
Yeah I realized this and had tried to correct. It seems like the global variable for $redis that is set in the configuration file for some reason is not persisted in the models at test run time. I could not find a solution, was thinking we should use a gem for a dummy redis in the tests, though that wouldn't be ideal.

**Comment by keighrim** [comment]
closed by #66
